styles/_header.scss

```diff
@@ -1,117 +1,213 @@
+}
+
+.nav-wrapper .nav-item {
+  -ms-flex-align: center;
+  align-items: center;
+  border: 1px solid transparent;
+  border-radius: 6px;
+  color: $background_color;
+  display: flex;
+  font-family: $heading_font;
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 20px;
+  margin-left: 5px;
+  max-width: 42px;
+  overflow: hidden;
+  padding: 0 10px 11px;
+  position: relative;
+  transition: max-width 0.2s ease-in-out 0.25s, color 0.2s ease-in-out 0.25s;
+  text-decoration: none;
   white-space: nowrap;
 
-  &[aria-expanded="true"] {
-    background-color: #fff;
-    box-shadow: 0 0 4px 0 rgba(0, 0, 0, .15), 0 4px 10px 0 rgba(0, 0, 0, .1);
-    border: solid 1px #ddd;
-    right: 0;
-    left: 0;
-    top: $header-height;
-    z-index: 1;
-
-    > a {
-      display: block;
-      margin: 20px;
-    }
+  &:active,
+  &:hover,
+  &:focus {
+    background: #f5f6f7;
+    color: $text_color;
+    max-width: 500px;
+    text-decoration: none;
   }
 }
 
-.nav-wrapper {
-  a {
-    @include tablet {
-      display: inline-block;
-    }
+.nav-wrapper .nav-item,
+.nav-wrapper .nav-icon,
+.nav-wrapper .nav-label  {
+  display: inline-block;
+}
 
-    border: 0;
-    color: $link_color;
-    display: none;
-    font-size: 14px;
-    padding: 0 20px 0 0;
-    width: auto;
-
-    [dir="rtl"] & { padding: 0 0 0 20px;}
-
-    &:hover,
-    &:focus,
-    &:active {
-      background-color: transparent;
-      color: $link_color;
-      text-decoration: underline;
-    }
+.nav-wrapper .nav-label {
+  padding: 10px 0 0 5px;
+  white-space: nowrap;
+}
 
-    &.sign-in { display: inline-block; }
+.nav-wrapper .nav-icon {
+  height: 20px;
+  position: relative;
+  top: 5px;
+  width: 20px;
+}
+
+.nav-wrapper .nav-icon path {
+  fill: $text_color;
+}
+
+/*
+I may need this button depending on how I render the site navigation.
+.menu-button {
+  background: $background_color;
+  border: 1px solid $border_color;
+  border-radius: 6px;
+  color: $text_color;
+  cursor: pointer;
+  font-size: 16px;
+  height: 30px;
+  min-width: 30px;
+  width: 30px;
+  z-index: 4;
+}*/
+
+@media only screen and (min-width: 576px) {
+  .logo {
+    grid-gap: 15px;
+    gap: 15px;
+    -ms-grid-columns: auto;
+    grid-template-columns: auto;
   }
 
-  .hide-on-mobile {
-    @include mobile {
-      @include visually-hidden;
-    }
+  .logo span {
+    padding-left: 18px;
   }
+}
 
-  .menu-button {
-    @include tablet { display: none; }
-    background: none;
-    border: 0;
-    color: $link_color;
-    display: inline-block;
-    margin-right: 10px;
-    padding: 0;
-    width: auto;
-
-    .icon-menu {
-      vertical-align: middle;
-      width: 13px;
-      height: 13px;
-    }
+@media only screen and (min-width: 992px) {
+  .header-area {
+    -ms-grid-columns: 310px 1fr max-content;
+    grid-template-columns: 310px 1fr max-content;
+  }
 
-    [dir="rtl"] & {
-      margin-left: 10px;
-      margin-right: 0;
-    }
+  .logo svg {
+    max-width: none;
```

Comment:
The logo seems a bit large. The dev-site has a max-width of none at this breakpoint, but it also has a container that is capping how big it can actually be. Effectively, the maximum size that the logo on the dev site can be is 162px.